CommonMark/tests/unit_tests.py

```diff
@@ -31,6 +31,14 @@ def test_null_string_bug(self):
             '<blockquote>\n<pre><code>sometext\n</code></pre>'
             '\n</blockquote>\n')
 
+    def test_dumpAST_orderedlist(self):
+        md = '1.'
+        ast = Parser().parse(md)
+        try:
+            CommonMark.dumpAST(ast)
+        except TypeError:
+            self.fail('TypeError')
```

Comment:
The test will fail if any exceptions are raised, so this try/except clause is unnecessary. Just change it to `CommonMark.dumpAST(ast)`